anvill/include/anvill/Program.h

```diff
@@ -239,6 +239,14 @@ class Program {
       const std::string &name,
       std::function<bool(const FunctionDecl *)> callback) const;
 
+  // Returns a possible control flow redirection for the given address
+  // or the input address itself if nothing is found
+  bool GetControlFlowRedirection(std::uint64_t &destination,
```

Comment:
`TryGet...` I like the `bool`-returning things to have `Try` prefix to convey in the name that it can fail.